frontend/src/components/contacts.tsx

```diff
@@ -1,136 +1,138 @@
-import React, { useState } from "react";
-import axios from "axios";
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
really would recommend not ignoring this one - why would you want to declare variables that you don't ever use?